src/database/models/project_item.rs

```diff
@@ -348,16 +348,6 @@ impl Project {
         if let Some(project) = project {
             Project::clear_cache(id, project.inner.slug, Some(true), redis).await?;
 
-            sqlx::query!(
```

Comment:
I just happened to notice this while reading this code - this is done again beneath the delete from mods_gallery. Am I missing something, or was this just unnecessary?

Reply:
yeah you can remove it if it's done twice